snippets/javascript/nuxt/nuxt.json

```diff
@@ -0,0 +1,87 @@
+{
+  "nuxt-nuxtlink": {
+    "prefix": ["nuxt nuxtlink", "nuxtlink"],
+    "body": ["<nuxt-link $1>$2</nuxt-link>$0"],
```

Comment:
Does it make sense to have separate <nuxt-link> without any attributes and nuxt-link with `to` attribute?
`<nuxt-link to=\"$1\">`
I guess by default users mostly will use the version with `to` attribute, so probably no need to have two versions, since if there is a rare case to use it without to, users will just remove that part, or if you think it is not a rare, we can leave it as `nuxt link empty` or something like this.

Reply:
I didn't think so also but I saw that it was done in vue.json so I figured there might be a special reason for it (that I haven't encountered yet). I will delete that now.